_tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md

```diff
@@ -0,0 +1,235 @@
+   node.attr.remote_store.repository.my-repo-1.settings.region: us-east-1
+   
+   # Translog repository settings
+   node.attr.remote_store.repository.my-repo-2.type: s3
+   node.attr.remote_store.repository.my-repo-2.settings.bucket: <Bucket Name 2>
+   node.attr.remote_store.repository.my-repo-2.settings.base_path: <Bucket Base Path 2>
+   node.attr.remote_store.repository.my-repo-2.settings.region: us-east-1
+   
+   # Enable Remote cluster state cluster setting
+   cluster.remote_store.state.enabled: true
+   
+   # Remote cluster state repository settings
+   node.attr.remote_store.repository.my-remote-state-repo.type: s3
+   node.attr.remote_store.repository.my-remote-state-repo.settings.bucket: <Bucket Name 3>
+   node.attr.remote_store.repository.my-remote-state-repo.settings.base_path: <Bucket Base Path 3>
+   node.attr.remote_store.repository.my-remote-state-repo.settings.region: <Bucket region>
+   
+   ```
+1. Stop the node you are migrating. Do not delete the volume associated with the container when you delete the container. The new OpenSearch container will use the existing volume. **Deleting the volume will result in data loss**.
+1. Deploy a new container running the same version of OpenSearch and mapped to the same volume as the container you deleted.
+1. Query the `_cat/nodes` endpoint after OpenSearch is running on the new node to confirm that it has joined the cluster. Wait for the cluster to become green again.
+1. Repeat steps 2 through 5 for each node in your cluster. 
+1. Reenable shard replication:
+   ```json
+   PUT "/_cluster/settings?pretty"
+   {
+       "persistent": {
+           "cluster.routing.allocation.enable": "all"
+       }
+   }
+   ```
+   The response should look similar to the following example:
+   ```json
+   {
+     "acknowledged" : true,
+     "persistent" : {
+       "cluster" : {
+         "routing" : {
+           "allocation" : {
+             "enable" : "all"
+           }
+         }
+       }
+     },
+     "transient" : { }
+   }
+   ```
+1. Confirm that the cluster is healthy:
+   ```bash
+   GET "/_cluster/health?pretty"
+   ```
+   The response should look similar to the following example:
+   ```json
+   {
+     "cluster_name" : "opensearch-dev-cluster",
+     "status" : "green",
+     "timed_out" : false,
+     "number_of_nodes" : 4,
+     "number_of_data_nodes" : 4,
+     "discovered_master" : true,
+     "active_primary_shards" : 1,
+     "active_shards" : 4,
+     "relocating_shards" : 0,
+     "initializing_shards" : 0,
+     "unassigned_shards" : 0,
+     "delayed_unassigned_shards" : 0,
+     "number_of_pending_tasks" : 0,
+     "number_of_in_flight_fetch" : 0,
+     "task_max_waiting_in_queue_millis" : 0,
+     "active_shards_percent_as_number" : 100.0
+   }
+   ```
+1. Clear the `remote_store.compatibility_mode` to not allow non-remote nodes to join back the cluster and `migration.direction` as well.
```

Comment:
```suggestion
1. Clear the `remote_store.compatibility_mode` and `migration.direction` settings so that non-remote nodes are not allowed to join the cluster by using the following command:
 
```